docs/dev/build-instructions/cmake_options.md

```diff
@@ -189,6 +189,10 @@ Build RNNoise for machine learning noise reduction
 Build the server (Murmur)
 (Default: ON)
 
+### snapshots
+Build a heavily optimized version, specific to the machine it's being compiled on.
+(Default: OFF)
+
```

Comment:
That's wrong, isn't it? :confused:

Reply:
I have no idea what happened there...
Maybe too much holiday cheer. :)

Reply:
Hehe ^^
Should be fixed by re-running the3 documentation script though :point_up:

Reply:
I'm not sure which script that is, but I will investigate.

Reply:
It's `scripts/generate_cmake_options_docs.py`

Reply:
I ran the script.